sample-app/src/hooks/useAutocomplete.tsx

```diff
@@ -0,0 +1,19 @@
+import { useRef, useState } from "react";
+import { AutocompleteResult, useAnswersActions } from '@yext/answers-headless-react';
+
+export function useAutocomplete(isVertical: boolean): [AutocompleteResult[], () => Promise<void>] {
+  const answersActions = useAnswersActions();
+  const autocompleteNetworkIds = useRef({ latestRequest: 0, responseInState: 0 });
+  const [ autocompleteResults, setAutoCompleteResults ] = useState<AutocompleteResult[]>([]);
```

Comment:
we have an item to be consistent about the capitzliation for AutoComplete vs Autocomplete but just making a note here

Reply:
oh yea, good catch. I made some changes to headless to have it lowercase, missed it here